ctrlf.el

```diff
@@ -74,6 +74,11 @@ return nil."
 Otherwise, the match count is only shown in the minibuffer."
   :type 'boolean)
 
+(defcustom ctrlf-go-to-end-of-match t
```

Comment:
```suggestion
(defcustom ctrlf-go-to-end-of-match nil
```

I'd like to leave the default as it currently stands. We should make backwards incompatible changes to default behavior only when clear demand has been demonstrated.